admin_manual/installation/source_installation.rst

```diff
@@ -2,47 +2,177 @@
+   provided by Redis are useful even in smaller deployments. For this reason they are recommended for
+   all use cases. However, they are not required for initial installation, and  are easy to integrate 
+   afterward when/if desired.
+
+The Nextcloud ``.tar.bz2`` archive contains the source code for Nextcloud Server itself as well as 
+the default apps and third-party libraries needed to run Nextcloud. 
+
+This document provides a walk-through for installing Nextcloud on an Ubuntu instance (Server-edition /
+LTS), with Apache and MariaDB, using the downloadable Nextcloud Server `Archive file 
+<https://nextcloud.com/install/>`_. This is the recommended method for installing Nextcloud from 
+scratch.
+
+Additional example installation guides, based on this material, for specific \*NIX OS distributions are 
+also available:
+
+- :doc:`./example_ubuntu`
+- :doc:`./example_centos`.
+- :doc:`./example_openbsd`.
+
+Alternative Installation Methods
+--------------------------------
+
+If you prefer or require an installation method that is more "packaged" or automated than a standard
+Archive-based installation, there are alternative installation methods. 
+
+.. note:: Note that is some variance between installation methods in terms of functionality and 
+   requirements. Consult each installation method's official documentation for specifics.
+
+* Nextcloud All-in-One (aka AIO)
+   - An official installation method that is maintained by `Nextcloud GmbH <https://nextcloud.com/>`_ (the 
+     legal entity that employs many of the most active Nextcloud developers and contributors).
+   - A pre-designed and container-based full Nextcloud stack.
+   - Designed for ease of deployment and maintenance with most features included.
+   - Includes Office, a turnkey Backup solution, Imaginary (for previews of heic, heif, illustrator, 
+     pdf, svg, tiff and webp) and much more (many components are optional). Full feature list available
+     `here <https://github.com/nextcloud/all-in-one?tab=readme-ov-file#nextcloud-all-in-one>`_.
+   - Up to 100 users are free (though without any direct service or support or guarantees). Free community 
+     support is provided via the `Nextcloud Community Help Forum <https://help.nextcloud.com/>`_ and a 
+     `dedicated GitHub Discussions area <https://github.com/nextcloud/all-in-one/discussions>`_ (on a best 
+     efforts basis).
+   - For slightly larger installations (>100 users) an AIO specific `Nextcloud Enterprise arrangement 
+     <https://nextcloud.com/all-in-one/>`_ is required (it includes a support SLA, security information and 
+     all the other benefits that come with `Nextcloud Enterprise <https://nextcloud.com/all-in-one/>`_).
+   - `Official Documentation <https://github.com/nextcloud/all-in-one>`_ (**Note**: see Official Documentation 
+     for AIO specific installation and usage details).
+* Nextcloud Community Snap
```

Comment:
I think there was one important limitation, that some configuration cannot be changed (not really sure what it was exactly, like adding new php modules for some apps).

Reply:
Yes things like that might be good to know in advance.